doc/index.rst

```diff
@@ -181,6 +181,31 @@ This represents the Tailwind configuration file:
     symfonycasts_tailwind:
         config_file: 'tailwind.config.js'
 
+If you include any other files containing classes to be detected outside the 
+default directories, like the `form themes provided with the Twig Bridge component <https://github.com/symfony/symfony/blob/HEAD/src/Symfony/Bridge/Twig/Resources/views/Form/tailwind_2_layout.html.twig/>`_, 
```

Comment:
I think we should link the docs instead, how about this link? 
```suggestion
default directories, like the ``symfony/twig-bridge``'s `Tailwind CSS form theme <https://symfony.com/doc/current/form/tailwindcss.html>`_
```

Reply:
That's better indeed, didn't know there was a dedicated page for this, or didn't find it!